### PR TITLE
feat: add the Plotly element

### PR DESCRIPTION
**Pull request by tpatel**
- Enables creating interactive charts using the plotly library
- Fixed the MessageContent width to always fill the full width even if the message is short

### CODE REVIEW

File: backend/chainlit/langchain/callbacks.py [hunk 1]
```diff
@@ -344,14 +344,27 @@ def _build_llm_settings(
         return provider, settings
 
 
+DEFAULT_TO_IGNORE = ["RunnableSequence", "RunnableParallel"]
```
Comment (willydouhard) [inline backend/chainlit/langchain/callbacks.py:347]: we shoud add these changes in this PR.

File: libs/components/package.json [hunk 2]
```diff
@@ -67,10 +67,12 @@
     "formik": "^2.4.3",
     "lodash": "^4.17.21",
     "mui-chips-input": "2.0.0",
+    "plotly.js": "^2.27.0",
     "react-dropzone": "^14.2.3",
     "react-file-icon": "^1.3.0",
     "react-markdown": "^8.0.7",
     "react-password-checklist": "^1.5.0",
+    "react-plotly.js": "^2.6.0",
```
Comment (willydouhard) [inline libs/components/package.json:75]: do we need both the dependencies?

File: libs/components/src/messages/components/MessageContent.tsx [hunk 3]
```diff
@@ -122,7 +122,7 @@ const MessageContent = memo(
 
     return (
       <Stack width="100%" direction="row">
-        <Box>
+        <Box width="100%">
```
Comment (willydouhard) [inline libs/components/src/messages/components/MessageContent.tsx:125]: why is this needed?

**Review by willydouhard — CHANGES_REQUESTED** [review]

File: libs/components/package.json (hunk 2 above)
Comment (tpatel) [reply libs/components/package.json:75]: yes, `plotly.js` is a peer dependency of `react-plotly.js`.

File: libs/components/src/messages/components/MessageContent.tsx (hunk 3 above)
Comment (tpatel) [reply libs/components/src/messages/components/MessageContent.tsx:125]: Here is what happens without it: when you have a short message, the container doesn't fill the full message column, so the elements are squeezed (the elements only have `max-width`, no `min-width` nor `width`).

image

Comment (tpatel) [reply libs/components/src/messages/components/MessageContent.tsx:125]: With the change:

image

**Review by willydouhard — CHANGES_REQUESTED** [review]
remove prisma dependencies?

**Comment by tpatel** [comment]
Thanks for the fix, I think the PR is ready now 👌
